### DISCUSSION TITLE
Divergence Issue of NREL 5MW Baseline Wind Turbine Under  Inflow Conditions with High Wind Speed

### DISCUSSION THREAD

**Discussion by Hitszh11** [OpenFAST/openfast#2104] [Q&A]
We have recently been running simulations using fast.farm.The incoming wind is set to a bts file generated by turbsim.According to the specifications of the NREL 5MW turbine, the range of wind speeds for the incoming wind is limited to the cut-in and cut-out speeds,which are 3 m/s and 25 m/s,respectively.So we set the reference wind speed range at hub height to 3 to 25 meters per second Turbulent intensities are set from 2% to 26 % with an interval of 3 %.The distribution of wind speed along the height direction is based on an exponential law.PLExp's range is from 0.1 to 0.26 with an interval of 0.04.The turbines are placed near the center of the wind field in a free-flow condition. When running the fast simulation with multiple wind field files as input using the above parameter combinations, we found that the program often reported errors and even aborted at wind speeds greater than 17 m/s.See the picture 1 attachment for the error reporting.In the simulation results after an error, the Rotspeed rises rapidly from the initial value and then drops back to zero, and then fluctuates at zero(as shown in picture 2).This error does not occur in a particular wind simulation, but randomly at wind speeds greater than 17 m/s.The cases where the ermor hasoccurred are shown in the picture 3.Can you tell me why this problem occurs and what is the effective solution?
![1](https://github.com/OpenFAST/openfast/assets/156392141/4a5b23aa-4e6e-4e0e-a319-39f31e9f763c)
![2](https://github.com/OpenFAST/openfast/assets/156392141/03f8304f-0953-4b6c-bfdd-6d6458f937c7)
![3](https://github.com/OpenFAST/openfast/assets/156392141/95213f47-6c3c-4b0a-a311-c381901bdcc3)

**Comment by jjonkman** [comment]
Dear @Hitszh11,

For each mean wind speed you are simulating, are you setting proper initial conditions for the rotor speed (`RotSpeed` in ElastoDyn) and blade-pitch angles (`BlPitch`(1-3) in ElastoDyn) based on their expected (mean) values, as has been discussed on many other issues and forum posts?  Otherwise, I would expect that the controller is not responding properly.

Best regards,

**Reply by Hitszh11** [reply]
Thank you for your reply. I made changes to the initial values of the rotor speed and  the blade-pitch angles in these problematic cases , and now the problem is solved!
